pkgs/test/test/runner/solo_test.dart

```diff
@@ -2,11 +2,10 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'package:test/test.dart';
```

Comment:
Why move this? I'm guessing it broke the `@TestOn` annotation behavior for this file.

Reply:
Unintentional. VS Code's auto sort feature. I'll need to clean this up.